app/lib/searching.rb

```diff
@@ -55,8 +75,85 @@ def search_left_list_for_link(link)
     end
```

Comment:
Read and then review lines 72..75 of this file 
https://apidock.com/ruby/Array/index

Reply:
it now takes up one line and I feel silly :)